### ISSUE TITLE
Use HTML details where appropiate

### ISSUE THREAD

**Issue by rullzer** [nextcloud/server#14379]
I came across https://dev.to/isfotis/html-details-is-awesome-1c20 today.
It would probably make sense we use this where possible (comments I guess?, maybe other places).
The awesome thing is accessibility comes for free on this.

CC: @nextcloud/designers @nextcloud/accessibility

**Comment by tyrylu** [comment]
Good idea. Not sure where i'd want to see this soonest, but yes, it can help accessibility in an interesting and useful way.

**Comment by jancborchardt** [comment]
Woah, that’s a great find! We could use it in places like:

- Collapsed comments as you mentioned
- Activities when there’s more activities in a block
- Installation for the "Storage & database" expandable block
- Files in the left navigation for "Shares" and "Favorites"?
- Any kind of collapsed folders in the left navigation actually?

Does this all make sense @nextcloud/accessibility?

**Comment by MorrisJobke** [comment]
Similar to https://dev.to/winduptoy/a-javascript-free-frontend-2d3e

**Comment by tyrylu** [comment]
That looks like a list of great ideas.
